web/wp-content/themes/dctx-v2/template-parts/content-blocks/block-wds-cta.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-nxfvdg.php, but found phpcs-scan-nxFVdg.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).